### ISSUE TITLE
t/pod-htmlify.t fails

### ISSUE THREAD

**Issue by moritz** [perl6/doc#56]
doc doesn't declare a dependency on Pod::To::HTML or URI because most users don't need to run htmlify. But `t/pod-htmlify.t` uses `Pod::Htmlify`, which in turn uses `URI::Escape`. Which makes the test fail when one tries to install doc via panda, and also [in the smoke test](http://smoke.perl6.org/project/p6doc)

**Comment by paultcochrane** [comment]
Sorry about that!  I've managed to skip `t/pod-htmlify.t` if `URI::Escape` isn't installed in ab78bc1, however I'm fairly sure there is an easier way to implement this than the way I used.  Any comments or advice as to how to improve the implementation are welcome.

@moritz I've tested on my box that the test runs when expected, however it might be a good idea to confirm this before closing this issue.
